### PR TITLE
Added torch.bitwise_not docstr

### PR DESCRIPTION
**Pull request by pbelevich**
Fixing #23311

### CODE REVIEW

**Comment by gchanan** [comment]
@zou3519 can you take a look?

File: torch/_torch_docs.py [hunk 1]
```diff
@@ -745,6 +745,23 @@ def merge_dicts(*dicts):
     tensor([0.0000, 0.0000, 0.0000, 1.0000, 1.0000, 0.0000, 0.5000])
 """)
 
+add_docstr(torch.bitwise_not,
```
Comment (zou3519) [inline torch/_torch_docs.py]: Needs entry in https://github.com/pytorch/pytorch/blob/master/docs/source/torch.rst

File: torch/_torch_docs.py [hunk 2]
```diff
@@ -745,6 +745,23 @@ def merge_dicts(*dicts):
...
+           r"""
+bitwise_not(input, out=None) -> Tensor
+
+Computes the bitwise NOT of the given input tensor. The input must be of
+integer or Boolean types.
```
Comment (zou3519) [inline torch/_torch_docs.py]: Maybe `integral` is a better word than `integer`? `bitwise_not` works on `torch.int` and `torch.long`, right?

**Review by zou3519 — APPROVED** [review]
lgtm

Did you check that it renders correctly on a local build?

**Comment by pbelevich** [comment]
@zou3519 yes, I did
`<p>See <a class="reference internal" href="torch.html#torch.bitwise_not" title="torch.bitwise_not"><code class="xref py py-func docutils literal notranslate"><span class="pre">torch.bitwise_not()</span></code></a></p>`

**Pull request closed without merging**